utility/importer/chrome_importer.cc

```diff
@@ -478,16 +478,6 @@ void ChromeImporter::ImportPasswords(
       }
     }
   }
-  std::vector<password_manager::PasswordForm> blocklist;
-  success = database.GetBlocklistLogins(&blocklist);
```

Comment:
are you sure about this fix? AFAIR locklist logins are basically entries that are saved when you refuse to save login/pass on a given website, the important part is basically an url where the user doesn't want to save passwords. So these entries are not actually useless

maybe we want to add an `if` to filter bad forms to prevent a check in the password store?

Reply:
Thanks! I didn't know that. Changed to filtering only invalid blocked forms.
Also updated login data. It has one more invalid blocked entry.
As invalid one is filtered, existing `ChromeImporterTest.ImportPasswords` should be passed.
W/o filtering, that test failed.